src/pages/RepoPage/CoverageOnboarding/GitHubActions/GitHubActions.tsx

```diff
@@ -208,11 +190,51 @@ jobs:
           slug: ${owner}/${repo}`
               : ''
           }
-`,
+  `,
     },
   }
 
-  const [framework, setFramework] = useState<Framework>('Jest')
+  return (
+    <div className="flex flex-col gap-6">
+      <Step1
```

Comment:
Step 1 header copy looks a little different on the figma:

Generate and upload coverage reports in your CI vs. Output a Coverage report file in your CI

Reply:
Also some minor copy differences with the subtext too